### PR TITLE
Extract images with webpack

### PR DESCRIPTION
**Pull request by maukoquiroga**
As images won't be required using JavaScript, webpack won't extract them
unless we explicitly tell him to do so.

### CODE REVIEW

**Comment by l-vincent-l** [comment]
What's the point of this PR ?

**Comment by maukoquiroga** [comment]
Avoid having to add en exception in .gitignore each time we want to add an image.

**Comment by l-vincent-l** [comment]
Can't we just ignore a path?

File: client/webpack.config.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 const { resolve }       = require('path')
 const webpack           = require('webpack')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const extractImages     = new CopyWebpackPlugin([{ from: 'images', to: '../images' }])
```
Comment (l-vincent-l) [inline client/webpack.config.js]: Are you sure of the destination path?

**Comment by maukoquiroga** [comment]
So you would be favorable to just avoid webpack and use another asset manager for fonts/images ? I agree

**Pull request merged**